.github/workflows/pkgdown.yaml

```diff
@@ -18,18 +18,13 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@master
 
-      - name: Install dependencies
-        run: |
-          install.packages("remotes")
-          remotes::install_deps(dependencies = TRUE)
-          install.packages("pkgdown")
-        shell: Rscript {0}
-
-      - name: Install package
-        run: R CMD INSTALL .
```

Comment:
Does R dependencies automatically installs the package?

Reply:
Yes, `local::.` in `extra-packages:` bellow will automatically install the local package, if `DESCRPTION` is in the current directory. Though, we might temporarily revert 226c17609f6c4eba389260d9ffec6daec69a37e8 until https://github.com/r-lib/pak/issues/358 gets fixed; it's random and likely due to an R session crash due to a dependency (could be one that needs compilation from C/C++ source).